app/config/services.yml

```diff
@@ -31,7 +31,7 @@ services:
     app.form.type.datetimepicker:
```

Comment:
But why? It still use in https://github.com/bocharsky-bw/symfony-demo/blob/form-type/src/AppBundle/Form/PostType.php#L52-L54